test/verify/check-machines

```diff
@@ -243,6 +243,18 @@ class TestMachines(MachineCase):
         if args["logfile"] is not None:
             wait(lambda: "Linux version" in self.machine.execute("cat {0}".format(args["logfile"])))
 
+        # Wait for the system to completely start
+        if args["logfile"] is not None:
+            wait(lambda: "login as 'cirros' user." in self.machine.execute("cat {0}".format(args["logfile"])))
+
+        # click the Send Non-Maskable Interrupt button (no change in VM state is expected)
+        b.click("#vm-subVmTest1-off-caret")
+        b.wait_visible("#vm-subVmTest1-sendNMI")
+        b.click("#vm-subVmTest1-sendNMI")
+
+        if args["logfile"] is not None:
+            wait(lambda: "NMI received" in self.machine.execute("cat {0}".format(args["logfile"])))
```

Comment:
@stefwalter Line #256 is not working for me and It failed the whole script. 

It seems that line #253 for sending the NMI: b.click("#vm-subVmTest1-sendNMI")" does work and send the NMI as required (because when removing this line #256 and manually checking, then the "..NMI received for unknown reason.." message appears in "logfile" as required). 

The problem here seems to be related to this VM serial console "logfile". 
When replacing line #256 with any other m.execute(...) command on this "logfile", for example when running:
m.execute("cat /var/log/libvirt/console-subVmTest1.log")
instead of line #256, I saw that the ".NMI received for unknown" message doesn't appear in that "logfile" while does appear in the graphic VM's console.

Do you have an idea why we have this problem? 
Thanks